spec/unit/mutest/integration_spec.rb

```diff
@@ -11,14 +11,15 @@
   end
 
   describe '.setup' do
-    subject { described_class.setup(kernel, name) }
+    subject { described_class.setup(requirer, name) }
 
-    let(:name)   { 'null'               }
-    let(:kernel) { class_double(Kernel) }
+    let(:name) { 'null' }
```

Comment:
I realize it might seem a little silly, but this project currently aligns the braces of single line lets. I plan to add this as a disabled cop with autocorrect for `rubocop-rspec` (or just in my personal http://github.com/backus/rubocop-devtools) if I want to enforce that longterm. Mind humoring me for now?